voucher/src/main/java/org/devcourse/voucher/console/Console.java

```diff
@@ -0,0 +1,21 @@
+package org.devcourse.voucher.console;
+
+import java.util.Scanner;
+
+public class Console implements Input, Output{
+    public static final String CREATE = "create";
+    public static final String LIST = "list";
+    public static final String EXIT = "exit";
+    private final Scanner sc = new Scanner(System.in);
+
+    @Override
+    public void printMenu() {
+        System.out.print("== Voucher Program ==\nType exit to exit the program.\nType create to create a new voucher.\nType list ot list all vouchers.\n>> ");
```

Comment:
명령어에 대한 정보가 상수로 관리되고 있는 데 여기에 사용 안되는 게 아쉬워요!

Reply:
```suggestion
        String menuString = """
            == Voucher Program ==
            Type exit to exit the program.
            Type create to create a new voucher.
            Type list ot list all vouchers.
        """.trim();
        System.out.print(menuString);
```
텍스트 블록을 사용할 수도 있고 다른 방법으로도 해당 문자열을 더 가독성 있게 표현해낼 수 있을 것 같아서 고민해보시면 좋을 것 같아요